.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,55 +1,49 @@
-name: Report a bug
+name: Bug report
 description: Tell us about a bug or issue you may have identified in OUDS Android
-title: "Bug - <Write here the bug summary>"
+title: "[Bug]: "
 labels: [ "🐞 bug", "🔍 triage" ]
+projects: [ "⚛️ [OUDS] Project" ]
 assignees: [ "florentmaitre", "paulinea" ]
+type: bug
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
   - type: checkboxes
     attributes:
       label: Prerequisites
       description: Take a couple minutes to help our maintainers work faster.
       options:
-        - label: I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-android/issues) for duplicate or closed feature requests
+        - label: I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-android/issues) for duplicate or closed feature requests.
```

Comment:
This sentence always says `duplicate or closed feature requests` in all templates. But maybe it should be `duplicate or closed issues` instead?